nntrainer/tensor/cpu_backend/fallback/fallback_internal.cpp

```diff
@@ -26,7 +26,10 @@
     float y0;                                                                  \
     unsigned int i, j;                                                         \
     for (ci = 0; ci != cM; ci++) {                                             \
-      y0 = Y[ci * incY] * beta;                                                \
+      y0 = 0.0f;                                                               \
+      if (beta != 0.0f) {                                                      \
```

Comment:
1. This doesn't recognize -0.0f (0.0f != -0.0f) (ref: https://en.cppreference.com/w/cpp/numeric/math/FP_categories )
2. Do you want to ignore round off errors?